tests/scipy_stats_test.py

```diff
@@ -1129,5 +1129,30 @@ def scipy_mode_wrapper(a, axis=0, nan_policy='propagate', keepdims=None):
                             tol=tol)
     self._CompileAndCheck(lax_fun, args_maker, rtol=tol)
 
+  @jtu.sample_product(
+    [dict(shape=shape, axis=axis)
+      for shape in [(0,), (7,), (47, 8), (0, 2, 3), (10, 5, 21)]
+      for axis in [None, *range(len(shape))
+    ]],
+    dtype=jtu.dtypes.integer + jtu.dtypes.floating,
+    contains_nans=[False],
```

Comment:
If we're not going to use `contains_nans`, I'd just remove it here and below for simplicity.